src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -111,37 +138,31 @@ function setupScriptProcessor(recordBuffer, passSampleToVisualizers) {
 
 /**
  * Set events and define callbacks for recording start/stop events.
- * @param {AudioBuffer} recordBuffer Buffer of the current recording.
+ * @param {object} recordingProperties Buffer of the current recording.
  */
-function setupRecording(recordBuffer) {
-  const recordButton = document.querySelector('#record');
-  const recordText = recordButton.querySelector('span');
-  const player = document.querySelector('#player');
-  const downloadButton = document.querySelector('#download');
-
-  async function prepareClip() {
+function setupRecording(recordingProperties) {
+  async function prepareClip(finalRecordBuffer) {
```

Comment:
Can we take this function out of `setupRecording` and add comments?

Reply:
Yes, it is moved.